frontend/app/src/connection/ConnectionManager.ts

```diff
@@ -75,10 +75,10 @@ interface Props {
   resetHostAuthToken: () => void
 
   /**
-   * Function to set the list of origins that this app should accept
-   * cross-origin messages from (if in a relevant deployment scenario).
+   * Function to set the host config for this app (if in a relevant deployment
+   * scenario).
    */
-  setAllowedOriginsResp: (resp: IAllowedMessageOriginsResponse) => void
+  setAllowedOrigins: (resp: IHostConfigResponse) => void
```

Comment:
This feels awkward given that the function name and type it takes are mismatched (note this is related to my other comments on the `setAllowedOrigins` function defined in `HostCommunicationManager`.) 

It's also confusing that this `setAllowedOrigins` is what calls `setHostConfig` in `App.tsx`, but the allowed origins list is a part of the host config, so things seem inverted.

Reply:
Agreed 👍 I renamed this to `onHostConfigResp` to better match what it is happening in this function.